keylkgSup/keylkg.db

```diff
@@ -289,6 +289,39 @@ record(ai, "$(P)VALUE:OUTPUT:2")
     info(archive, "VAL")
 }
 
+record(seq, "$(P)SCAN")
+{
+    field(DOL0, "$(P)SCAN:SP.RVAL")
+    field(DOL1, "$(P)SCAN:SP.RVAL")
+    field(DOL2, "$(P)SCAN:SP.RVAL")
+    field(DOL3, "$(P)SCAN:SP.RVAL")
+    field(LNK0, "$(P)OFFSET:OUTPUT:1.SCAN")
+    field(LNK1, "$(P)OFFSET:OUTPUT:2.SCAN")
+    field(LNK2, "$(P)MEASUREMODE:HEAD:A.SCAN")
+    field(LNK3, "$(P)MEASUREMODE:HEAD:B.SCAN")
+    field(SELM, "All")
+}
+
+record(mbbo, "$(P)SCAN:SP")
+{
+    field(OUT, "$(P)SCAN PP")
+    field(DTYP, "Raw Soft Channel")
+    
+    field(ZRVL, "4")
+    field(ONVL, "0")
+    
+    field(ZRST, "5 second")
+    field(ONST, "Passive")
+    
+    field(VAL, "0")
+    field(PINI, "YES")
+    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SCAN:SP")
+    field(SDIS, "$(P)DISABLE")
```

Comment:
Since this is a soft record I don't actually think you need these declarations - and removing them should allow us to run the corresponding ioc test in recsim as well as devsim (you can also then remove the sim record below).